go.mod

```diff
@@ -8,10 +8,11 @@ require (
 	github.com/fsnotify/fsnotify v1.7.0
 	github.com/google/renameio v1.0.1
 	github.com/gorilla/mux v1.8.1
+	github.com/mattn/go-sqlite3 v1.14.22
```

Comment:
For anybody wondering why this is in here, it's now a direct dependency due to the change in https://github.com/canonical/microcluster/pull/86 which imports the package for one of the tests in `internal/extensions/extensions_test.go`.